ctags/parsers/c.c

```diff
@@ -2234,12 +2232,12 @@ static boolean skipPostArgumentStuff (statementInfo *const st,
 	{
 		switch (c)
 		{
-			case ')':				break;
-			case ':': skipMemIntializerList (token);break;	/* ctor-initializer */
-			case '[': skipToMatch ("[]");			break;
-			case '=': cppUngetc (c); end = TRUE;	break;
-			case '{': cppUngetc (c); end = TRUE;	break;
-			case '}': cppUngetc (c); end = TRUE;	break;
+		case ')':                               break;
+		case ':': skipMemIntializerList (token);break;  /* ctor-initializer */
+		case '[': skipToMatch ("[]");           break;
+		case '=': cppUngetc (c); end = TRUE;    break;
+		case '{': cppUngetc (c); end = TRUE;    break;
+		case '}': cppUngetc (c); end = TRUE;    break;
```

Comment:
that one looks odd, but well.

though, I highly doubt we'll merge *c.c* from U-CTags, as we have *different* things in it, and that they have another C++ one anyway.  I'll probably try to extract the Vala part to a separate parser (I already started a while back IIRC, experimenting with other stuff in the way), and we'll "only" be left with like 3 parsers in there, that have probably diverged a lot less.
Though OTOH maybe yeah, when it's down to C#/Java/Vera/D it might start and be mergable.